src/main/ArrayVisualizer.java

```diff
@@ -529,21 +631,75 @@ private void drawStats(Color textColor, boolean dropShadow) {
+        // this.mainRender.drawString(this.statSnapshot.getSwapCount(),       xOffset, (int) (windowRatio * 210) + yOffset);
+        // this.mainRender.drawString(this.statSnapshot.getReversalCount(),   xOffset, (int) (windowRatio * 235) + yOffset);
+        // this.mainRender.drawString(this.statSnapshot.getMainWriteCount(),  xOffset, (int) (windowRatio * 275) + yOffset);
+        // this.mainRender.drawString(this.statSnapshot.getAuxWriteCount(),   xOffset, (int) (windowRatio * 300) + yOffset);
+        // this.mainRender.drawString(this.statSnapshot.getAuxAllocAmount(),  xOffset, (int) (windowRatio * 325) + yOffset);
+        // this.mainRender.drawString(this.statSnapshot.getSegments(),        xOffset, (int) (windowRatio * 350) + yOffset);
+        statLoop:
+        for (int statType : statsConfig) {
+            // System.out.println(yPos);
+            String stat;
+            switch (statType) {
+                case STAT_LINE_BREAK:
+                    yPos += 15;
+                    continue statLoop;
+                case STAT_SORT_IDENTITY:
+                    stat = statSnapshot.getSortIdentity();
+                    break;
+                case STAT_ARRAY_LENGTH:
+                    stat = statSnapshot.getArrayLength();
+                    break;
+                case STAT_FRAMERATE:
+                    stat = statSnapshot.getFramerate();
+                    break;
+                case STAT_SORT_DELAY:
+                    stat = statSnapshot.getSortDelay();
+                    break;
+                case STAT_VISUAL_TIME:
+                    stat = statSnapshot.getVisualTime();
+                    break;
+                case STAT_EST_SORT_TIME:
+                    stat = statSnapshot.getEstSortTime();
+                    break;
+                case STAT_COMPARISONS:
+                    stat = statSnapshot.getComparisonCount();
+                    break;
+                case STAT_SWAPS:
+                    stat = statSnapshot.getSwapCount();
+                    break;
+                case STAT_REVERSALS:
+                    stat = statSnapshot.getReversalCount();
+                    break;
+                case STAT_MAIN_WRITE:
+                    stat = statSnapshot.getMainWriteCount();
+                    break;
+                case STAT_AUX_WRITE:
+                    stat = statSnapshot.getAuxWriteCount();
+                    break;
+                case STAT_AUX_ALLOC:
+                    stat = statSnapshot.getAuxAllocAmount();
+                    break;
+                case STAT_SEGMENTS:
+                    stat = statSnapshot.getSegments();
+                    break;
+                default:
+                    stat = null; // Unreachable
+            }
+            mainRender.drawString(stat, xOffset, (int)(windowRatio * yPos) + yOffset);
+            yPos += 25;
```

Comment:
like you said, why not scale for the font size?

Reply:
Because I don't even know how I'd do it

Reply:
naively I'd just make it proportional but maybe fonts are more complicated